lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -177,12 +177,10 @@ swapNodes = (targetBody, existingNodes, options) ->
       else
         targetNode = targetNode.cloneNode(true)
         existingNode.parentNode.replaceChild(targetNode, existingNode)
-        if targetNode.nodeName == 'SCRIPT' && targetNode.getAttribute("data-turbolinks-eval") != "false"
-          executeScriptTag(targetNode)
```

Comment:
Weird. Just said that we should rerun scripts whenever we are replacing elements. 


> On Mar 8, 2015, at 11:35, Kasper Timm Hansen <notifications@github.com> wrote:
> 
> In lib/assets/javascripts/turbolinks.js.coffee:
> 
> > @@ -177,12 +177,10 @@ swapNodes = (targetBody, existingNodes, options) ->
> >        else
> >          targetNode = targetNode.cloneNode(true)
> >          existingNode.parentNode.replaceChild(targetNode, existingNode)
> > -        if targetNode.nodeName == 'SCRIPT' && targetNode.getAttribute("data-turbolinks-eval") != "false"
> > -          executeScriptTag(targetNode)
> @dhh your comment is empty.
> 
> —
> Reply to this email directly or view it on GitHub.
>